core/src/test/java/org/apache/calcite/test/SqlToRelConverterTest.java

```diff
@@ -645,6 +645,7 @@ public final Sql expr(String expr) {
     sql("select distinct sal + 5 from emp").ok();
   }
 
+  @Disabled
```

Comment:
??

Reply:
misplaced the handle, repositioned as necessary.